### DISCUSSION TITLE
Déploiement des fichiers de config

### DISCUSSION THREAD

**Discussion by atifmohamed** [MTESSDev/FRW#6] [Annonces]
Bonjour,

Dans ce diagramme [FRW-APIs-v2.pdf](https://github.com/MTESSDev/FRW/blob/main/FRW-APIs-v2.pdf), il est indiqué qu’un système autorisé pourrait déployer les fichiers de config (la définition des formulaires) via un pipeline en appelant l ’API du MESS.
Après vérification du [swagger](https://github.com/MTESSDev/FRW/tree/main/Swagger) de l’API, je n’ai pas trouvé la méthode à appeler pour le déploiement.
Mes questions :
Est-ce que cette méthode est publiée dans un autre API ?
L’autre question, est-ce que chaque environnement à son propre dépôt de ces fichiers de config ?
Merci.

**Comment by anisite** [comment]
Bonjour,

c'est une bonne question, l'API n'a effectivement pas été publiée "publiquement", mais les équipes utilisant Azure DevOps peuvent simplement ajouter l'extension FRW du marketplace https://marketplace.visualstudio.com/items?itemName=MTESS.mtess-frw-deploiement 

Voulez-vous déployer à partir d'une autre source ?

**Reply by atifmohamed** [reply]
Merci pour la réponse rapide et claire.
Dans mon équipe nous utiliserons Azure DevOps
